cms-contentful/it/ContentfulCmsServiceIT.java

```diff
@@ -0,0 +1,68 @@
+import io.commercetools.sunrise.cms.CmsIdentifier;
+import io.commercetools.sunrise.cms.contentful.ContentfulCmsService;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.TimeUnit;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ContentfulCmsServiceIT {
+    private static final Locale DE = Locale.forLanguageTag("de");
+    private static final Locale EN_US = Locale.forLanguageTag("en-US");
+    private static final List<Locale> SUPPORTED_LOCALES = asList(DE, EN_US);
+
+    // credentials for contentful demo account
+    private static final String SPACE_ID = "cfexampleapi";
+    private static final String TOKEN = "b4c0n73n7fu1";
+    private ContentfulCmsService contentfulCmsService;
+
+    @Before
+    public void setUp() throws Exception {
+        contentfulCmsService = ContentfulCmsService.of(SPACE_ID, TOKEN);
+    }
+
+    @Test
+    public void whenAskForExistingStringContentThenGet() throws Exception {
+        CmsIdentifier identifier = CmsIdentifier.ofEntryTypeAndKeyAndField("human", "Finn", "description");
+        CompletionStage<Optional<String>> optionalCompletionStage = contentfulCmsService.get(SUPPORTED_LOCALES, identifier);
+        Optional<String> content = optionalCompletionStage.toCompletableFuture().get(5, TimeUnit.SECONDS);
+
+        assertThat(content.isPresent()).isTrue();
+        assertThat(content.get()).isEqualTo("Fearless adventurer! Defender of pancakes.");
```

Comment:
As I was using contentful demo account, I've used their content.
I can create separate space in contentful for commercetools account, reuse here it's credenials
and implement contenfful-management to create it's content for it tests.
Can this be done later?
When I will finish with core implementation. It will be additional effort
and I would not expect it's demo content to change anytime soon.